### DISCUSSION TITLE
第5节课笔记(19班)

### DISCUSSION THREAD

**Discussion by LYMDLUT** [InternLM/Tutorial#332] [笔记]
第5节课笔记(19班)

**Comment by Jiangzzzzl** [comment]
https://blog.csdn.net/2301_80618119/article/details/135563114?spm=1001.2014.3001.5501

**Comment by Sixdes** [comment]
https://bbs.csdn.net/topics/617942093

**Comment by hu-qi** [comment]
实操了部署，量化还在学习中...
https://www.mdnice.com/writing/ba0baf36bb6341be9fbfadb907c08a6e

**Comment by romanwin28** [comment]
https://blog.csdn.net/George_han/article/details/135646624

**Comment by jts250** [comment]
https://zhuanlan.zhihu.com/p/678303369

**Comment by melonwine** [comment]
https://github.com/melonwine/InternLMPlay/blob/master/05/note05.md

**Comment by AllYoung** [comment]
https://blog.csdn.net/sunshine_youngforyou

**Comment by mkdirmushroom** [comment]
[第5节课笔记](https://github.com/mkdirmushroom/note/blob/main/InternLM/tutorial4_note.md)

**Comment by Hamasuta666** [comment]
第五节课笔记（作业在文末）：https://zhuanlan.zhihu.com/p/680990420
